cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -42,6 +43,8 @@
     'weber': 1635923188204,  # 2021-11-03 07:06:28.204 UTC
 }
 
+ZPHASE_DATA = {'rainbow': 'rainbow_zphase.json', 'weber': 'weber_zphase.json'}
```

Comment:
I am a little concerned about the naming scheme here.  Do you think we should put in an extra qualifier, such as 'rainbow_sample_zphase.json' or 'rainbow_average_zphase.json' in case we need to distinguish it from data from a specific day?

Reply:
Found and added dates for these.